analyzers/src/packaging/SonarAnalyzer.CSharp.nuspec

```diff
@@ -22,10 +22,10 @@
     </frameworkAssemblies>
   </metadata>
   <files>
-    <file src="bin\Release\SonarAnalyzer.CFG.dll" target="analyzers" />
-    <file src="bin\Release\SonarAnalyzer.dll" target="analyzers" />
-    <file src="bin\Release\SonarAnalyzer.CSharp.dll" target="analyzers" />
-    <file src="bin\Release\Google.Protobuf.dll" target="analyzers" />
+    <file src="..\SonarAnalyzer.CSharp\bin\Release\net46\SonarAnalyzer.CFG.dll" target="analyzers" />
```

Comment:
Can this whole directory be in `analyzers\packaging` instead of `analyzers\src\packaging`?

Reply:
Like that, we could have `analyzers\packaging\binaries` to keep the directory count in `analyzers` low